### PR TITLE
Enhancement: Update Ornatrix Docs

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
This PR enhances the artist and admin orantrix docs and moves ornatrix admin docs to a dedicated page.

## Testing notes:
Go through the changes and let me what do you think.

### CODE REVIEW

File: website/docs/addon_maya_ornatrix_admin.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#### Ornatrix Rig Loader
+
+Access the settings for the Ornatrix Rig Loader at:
+- `ayon+settings://maya/load/OxRigLoader`
+
+![](assets/maya/admin/ornatrix_loaders.png)
+
+1. Toggle the loader on or off.
+2. **Create Ornatrix Cache instance on load:**
+    Automatically create an Ornatrix Cache instance upon loading, akin to how an animation rig automatically generates an animation instance for publishing loaded character rigs—applied specifically for Ornatrix rigs.
```
Comment (moonyuet) [inline website/docs/addon_maya_ornatrix_admin.md:44]: Not sure the more concise to way to elaborate this, but.
```suggestion
    Ornatrix Cache instance is automatically created whenever an Ornatrix rig is loaded into the scene, akin to how the loaded character rig automatically generates the animation instance for publishing.
```

Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_admin.md:44]: me neither, let's summon @BigRoy .

File: website/docs/addon_maya_ornatrix_artist.md [hunk 2]
```diff
@@ -98,3 +68,18 @@ The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix dat
 ### Loading
 
 The **Ornatrix Cache** supports to load Ornatrix alembic (.abc), Vray Proxy (.abc) and Arnold Scene Source (.abc).
+
+## FAQ
+### How to connect an Ornatrix rig with a pointcache or animation?
+After loading the Ornatrix Rig and a pointcache to connect it to, you can use the AYON tools to connect it for you:
```
Comment (moonyuet) [inline website/docs/addon_maya_ornatrix_artist.md:74]:
```suggestion
Once you’ve loaded the Ornatrix Rig and its corresponding pointcache, you can connect them using the AYON tools: 
```

Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md:74]: what about 
```suggestion
Once you’ve loaded the Ornatrix Rig and its corresponding pointcache, you can connect them using AYON Inventory Actions: 
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 3]
```diff
@@ -98,3 +68,18 @@ The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix dat
...
+
+1. Go to `AYON` -> `Manage...`
+2. In the inventory select the assets from oxrig and animation/pointcache product type.
+3. Right-click `Actions` -> `Connect Ornatrix Rig`
+The Ornatrix geometry will be connected the animation rig with the transform and mesh attributes.
+
+:::tip
+The connecting will occur by node name - so if it fails, make sure the node name for the geometry is the same across your animation caches and the ornatrix exports.
```
Comment (moonyuet) [inline website/docs/addon_maya_ornatrix_artist.md:82]:
```suggestion
The connection depends on the name of node. If it fails, you need to make sure the node name for the geometry is consistent across your animation caches and the Ornatrix exports.
```

File: website/docs/addon_maya_ornatrix_artist.md [hunk 4]
```diff
@@ -98,3 +68,18 @@ The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix dat
...
+
+**Double geometry:** Be aware that both the ornatrix rig and your animated cache have the mesh geometry. After connecting the fur you will most likely want to hide one of the two to ensure you do not have two overlapping meshes during rendering.
```
Comment (moonyuet) [inline website/docs/addon_maya_ornatrix_artist.md:84]:
```suggestion
**Double geometry:** Be aware that your ornatrix rig and animated cache have the mesh geometry. After connecting the fur you will most likely want to hide one of the two to ensure you do not have two overlapping meshes during rendering.
```

File: website/docs/addon_maya_ornatrix_artist.md (hunk 2 above)
Comment (moonyuet) [reply website/docs/addon_maya_ornatrix_artist.md:74]: We should use you have instead of you've.

Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md:74]: what about: 
```suggestion
Once you load the Ornatrix Rig along with its corresponding pointcache, you can link them using AYON Inventory Actions:
```

File: website/docs/addon_maya_ornatrix_artist.md (hunk 3 above)
Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md:82]: what about: 
```suggestion
The connection relies on the node's name. If it fails, ensure that the geometry's node name is consistent between your animation caches and the Ornatrix exports.
```

File: website/docs/addon_maya_ornatrix_admin.md (hunk 1 above)
Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_admin.md:44]: what about: 
```suggestion
    When enabled, an Ornatrix Cache instance is automatically created each time an Ornatrix rig is loaded into the scene, similar to how loading a character rig automatically creates the animation instance for publishing.
```

File: website/docs/addon_maya_ornatrix_artist.md (hunk 3 above)
Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md]: I've applied the last suggestion. feel free to unresolve.
https://github.com/ynput/ayon-documentation/pull/237/commits/5df28b622d2dbf737f57502e6b492bda2b003790

File: website/docs/addon_maya_ornatrix_artist.md (hunk 2 above)
Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md]: I've applied the last suggestion. feel free to unresolve.
https://github.com/ynput/ayon-documentation/pull/237/commits/5df28b622d2dbf737f57502e6b492bda2b003790

File: website/docs/addon_maya_ornatrix_artist.md [hunk 5]
```diff
@@ -13,31 +14,14 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
...
-
-![Maya - Ornatrix Validator Setting](assets/maya/artist/ox_validator_maya_addon_setting.jpg)
-
-![Maya - Ornatrix Loader Setting](assets/maya/artist/ox_loaders_maya_addon_setting.png)
-
 ## Workflow
-AYON has two creators for Ornatrix publishing: ```Ornatrix Rig``` and ```Ornatrix Cache```.
+AYON has two creators for Ornatrix publishing: `Ornatrix Rig` and `Ornatrix Cache`.
 
 ## Ornatrix Rig
```
Comment (MustafaJafar) [inline website/docs/addon_maya_ornatrix_artist.md:20]: I've a question about Ornatrix rig. 

So, this section doesn't mention anything about connecting rigs to pointcaches or animations. 
and I can't understand why would artists need to connect them. 

Could we elaborate on this workflow?

Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md:20]: Also, test notes in this PR https://github.com/ynput/ayon-maya/pull/1 look useful. could we use bits of them to write about the workflow and what artists should do and expect ?

Comment (moonyuet) [reply website/docs/addon_maya_ornatrix_artist.md:20]: > I've a question about Ornatrix rigs.
> 
> So, this section doesn't mention anything about connecting rigs to pointcaches or animations. and I can't understand why would artists need to connect them.
> 
> Could we elaborate on this workflow?

it is similar to how the connect yeti rig works. When loading the animated characters and they are not at the original positions(e.g. they are sitting or running), the connect action can help the rigs to find the correct position of where the fur(e.g. hair) located.

Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md:20]: so, we link the animated character with its animated hair/fur/groom to be able to see the full animation.

Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md:20]: Thank you for your explanation. it helped me update the answer.
https://github.com/ynput/ayon-documentation/pull/237/commits/f218f5198a3ebd2a14154c0e1a5f6020bbbd50b0

File: website/docs/addon_maya_ornatrix_artist.md [hunk 6]
```diff
@@ -98,3 +68,19 @@ The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix dat
 ### Loading
 
 The **Ornatrix Cache** supports to load Ornatrix alembic (.abc), Vray Proxy (.abc) and Arnold Scene Source (.abc).
+
+## FAQ
+### How to connect an Ornatrix rig to a pointcache or animation?
+
+To guide Ornatrix rig to its proper location. you'll need to connect it to the animation, aligning both transform and mesh attributes.
```
Comment (moonyuet) [inline website/docs/addon_maya_ornatrix_artist.md:75]:
```suggestion
To position the Ornatrix rig correctly, you need to connect it to the animation, aligning both the transform and mesh attributes
```

File: website/docs/addon_maya_ornatrix_admin.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+At present, we support two Orantrix product types. You can let artists create these products by enabling them in the Maya create settings.
+
+- `ayon+settings://maya/create/CreateOxRig`
+- `ayon+settings://maya/create/CreateOxCache`
+
+![](assets/maya/admin/ornatrix_creators.png)
+
+### Validators
+
+Currently, we provide content validation for the two Orantrix product types.
```
Comment (BigRoy) [inline website/docs/addon_maya_ornatrix_admin.md:30]:
```suggestion
Currently, we provide content validation for the two Ornatrix product types.
```

File: website/docs/addon_maya_ornatrix_admin.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+You can enable these validations, set them as optional to let artists skip them when publishing, or deactivate them if needed.
+
+- `ayon+settings://maya/publish/ValidateOrnatrixRigContent`
+- `ayon+settings://maya/publish/ValidateOrnatrixCacheContent`
```
[the remark this answers is not in the document]
Comment (moonyuet) [reply website/docs/addon_maya_ornatrix_admin.md:34]: > Should something elaborate on WHAT these validate?
Both validators ensure only ornatrix-related nodes are included.

[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_admin.md:35]: added in https://github.com/ynput/ayon-documentation/pull/237/commits/7a84098e9fe5327cfc62278b0b720399bf206088

File: website/docs/addon_maya_ornatrix_admin.md (hunk 7 above)
Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_admin.md]: fixed in https://github.com/ynput/ayon-documentation/pull/237/commits/7a84098e9fe5327cfc62278b0b720399bf206088

File: website/docs/addon_maya_ornatrix_artist.md (hunk 6 above)
Comment (MustafaJafar) [reply website/docs/addon_maya_ornatrix_artist.md]: fixed in https://github.com/ynput/ayon-documentation/pull/237/commits/7a84098e9fe5327cfc62278b0b720399bf206088

**Pull request merged**